src/Main/Tasks.cpp

```diff
@@ -0,0 +1,48 @@
+#include "Tasks.h"
+
+#include <QDebug>
+#include <QJsonArray>
+
+#include "Settings.h"
+
+using namespace FOEDAG;
+
+Tasks* Tasks::m_instance = nullptr;
+Tasks::Tasks() { fprintf(stderr, "Tasks Created\n"); }
```

Comment:
why not std::cerr ?

Reply:
Just debug code i forgot to remove. Originally thought this PR was going to be more of a temp thing to look at the code instead of an actual check-in. Still getting used to the process ;)